src/libs/CardUtils.ts

```diff
@@ -72,11 +73,13 @@ function getYearFromExpirationDateString(expirationDateString: string) {
  * @param cardList - collection of assigned cards
  * @returns collection of assigned cards grouped by domain
  */
-function getDomainCards(cardList: Record<string, OnyxTypes.Card>) {
+function getDomainCards(cardList: OnyxCollection<OnyxTypes.Card>) {
```

Comment:
is this really a OnyxCollection? if not I think we should stick to just `Record<string, OnyxTypes.Card>`

Reply:
should it be just `OnyxEntry<OnyxTypes.CardList`?

Reply:
Yes, we receive this data from the Onyx